fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -156,18 +149,18 @@ RdfStream getTriples(final IdentifierConverter<Resource, FedoraResource> idTrans
                          final Set<? extends TripleCategory> contexts);
 
     /**
-     * Get the JCR Base version for the node
+     * Get the base version for the node
      *
      * @return base version
      */
-    public Version getBaseVersion();
+    FedoraResource getBaseVersion();
 
     /**
-     * Get JCR VersionHistory for the node.
+     * Get a stream of version labels associated with this resource
```

Comment:
I would rather not use the word "label" for that. It's an identifier, no? An identifier for a version within the context of a versioned resource, and it helps compose the identifier of that version outside that context.